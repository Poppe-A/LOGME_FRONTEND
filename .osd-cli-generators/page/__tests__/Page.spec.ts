const ejs = require('ejs');
const frontmatter = require('frontmatter');

describe('Page generator', () => {
    it('generates correctly', async () => {
        const output = await ejs.renderFile(`${__dirname}/../Page.tsx.ejs`, {
            props: {
                pascalCaseName: 'FoobarPage',
            },
        });
        const fmData = frontmatter(output);

        expect(fmData.content).toMatchSnapshot();
    });
});
