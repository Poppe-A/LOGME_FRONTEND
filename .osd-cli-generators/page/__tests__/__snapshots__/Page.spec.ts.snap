// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page generator generates correctly 1`] = `
"import React from 'react';
import styled from 'styled-components';
import { Helmet } from 'react-helmet-async';

const Container = styled.section\`
    display: flex;
    flex: 1;
    background-color: white;
    justify-content: center;
    align-items: center;
\`;

const FoobarPagePage: React.FC = () => {
    return (
        <>
            <Helmet>
                <title>FoobarPage!!!!</title>
            </Helmet>
            <Container data-testid=\\"FoobarPage_page\\">
                <span>This is the FoobarPage page. Its one of the best pages!</span>
            </Container>
        </>
    );
};

export default FoobarPagePage;
"
`;
